constants.go

```diff
@@ -57,9 +54,17 @@ const (
       <backend model='random'>/dev/urandom</backend>
     </rng>
 {{- range .ExtraDevices }}
-	{{ . }}
+    {{ . }}
 {{- end }}
   </devices>
 </domain>`
 	VSockDevice = `<vsock model='virtio'><cid auto='yes'/></vsock>`
 )
+
+func NetworkDevice(name string) string {
```

Comment:
slight preference for `networkName` here, or just `network` to be consistent with the field name ni `Driver`

Reply:
renamed!